lib/dialects/mysql/query.js

```diff
@@ -118,6 +130,30 @@ Query.prototype.formatResults = function(data) {
   return result;
 };
 
+Query.prototype.logWarnings = function (results) {
+  var self = this;
+  return this.run('SHOW WARNINGS')
+  .then( function(warningResults) {
```

Comment:
Please put the `then` on the previous line, remove space before `function` and add space after ;) (sorry for nitpicking)